### ISSUE TITLE
 LoadError:   cannot load such file -- spec_helper

### ISSUE THREAD

**Issue by atxrenegade** [learn-co-students/rails-activerecord-models-and-rails-readme-v-000#103]
LESSON STATES:  "Let's create a new file: spec/models/post_spec.rb. In that file, place the following code:

require 'rails_helper'
 
describe Post do
 
end"

Results in: 
"An error occurred while loading ./spec/models/post_spec.rb.
Failure/Error: require 'rails_helper'

LoadError:
  cannot load such file -- rails_helper"

Tests will not load correctly until the following line is removed or commented out:
#require 'rails_helper'

**Comment by mddilley** [comment]
I just encountered this same error, and the lesson states:

`# Finally, we created the initial RSpec config:`

`rails g rspec:install`

The code in this repository does not reflect the changes made by this command and needs to be amended. The lesson could also be amended to instruct the student to run this command in the console prior to the next step.

Running this command in the console creates the spec folder and rails_helper.rb within it (along with some other files). Then, the lab can be completed as intended.

**Comment by meg-gutshall** [comment]
I second @mddilley's comment. Could someone please address this 10-week-old open issue??

**Comment by guysbryant** [comment]
I have created a pull request which fixes this issue if somebody would like to approve it.

**Comment by Sdcrouse** [comment]
I think this issue has been fixed. When I opened this lab, the `spec` folder contained these files: `rails_helper.rb` and `spec_helper.rb`. I was able to run the tests and complete the lab without any trouble. That said, there were quite a few deprecation warnings (I'm using Rails version 5.0.7.2).

**Comment by meg-gutshall** [comment]
Yes @Sdcrouse, it looks like @drakeltheryuujin addressed this in July with commit e6608aea1c72fa5c3eef912a43d02f7aa49bcef2 and may have just forgotten to close out this issue. Thanks for following up on this though!  YOU --> 💯 👍

**Comment by Sdcrouse** [comment]
You're welcome, @meg-gutshall ! 😁
